examples/web_socket/stock-ticker.cpp

```diff
@@ -28,6 +28,8 @@ static constexpr uint16_t default_port = 8080;
 
 static constexpr size_t default_max_connections = 128;
 
+static constexpr caf::timespan default_interval = caf::timespan{1s};
```

Comment:
Shouldn't it work without the type repetition?

```suggestion
static constexpr caf::timespan default_interval = 1s;
```

Reply:
Thanks, this has been updated in the PR.